### ISSUE TITLE
How can I set an optimizer along with a learning rate(Keras)?

### ISSUE THREAD

**Issue by byshichen** [hyperopt/hyperopt#310]
like this -> optimizers.RMSprop(lr = 1e-3, epsilon=1e-6)

How can I set this in the search space though?

Cheers

**Comment by aditsanghvi94** [comment]
This is what I did. 
def f_nn(params):   
    print ('Params testing: ', params)
    optimizer = Adam(10**params['lr'])
    # build model
    model.compile(loss='binary_crossentropy', optimizer=optimizer, metrics=['accuracy'])
